Sushmitha/c program files/program3.c

```diff
@@ -0,0 +1,25 @@
+//3. write a program in c to count a total number of duplicate elements in an array
+
+#include <stdio.h>
+int main()
+{
+	int arr[5], i, j, Count = 0;
+    printf("Enter 5 element of an array : \n");
+	for (i = 0; i < 5; i++)
+	{
+    	scanf("%d", &arr[i]);
+   	}
+	for (i = 0; i < 5; i++)
+	{
+		for(j = i + 1; j < 5; j++)
+		{
+    		if(arr[i] == arr[j])
+    		{
+    			Count++;
+				break;
```

Comment:
![image](https://user-images.githubusercontent.com/30323270/65419494-62d3a980-de1c-11e9-866e-5b71523bb19d.png)

How number of duplicate elements are 3 ??

Reply:
because 3 ones are there. so duplicate element value is 3.